examples/api/date.py

```diff
@@ -7,13 +7,14 @@
 formatters.  See major_minor_demo1.py for more information on
 controlling major and minor ticks
 
-All matplotlib date plotting is done by converting date instances into
-days since the 0001-01-01 UTC.  The conversion, tick locating and
-formatting is done behind the scenes so this is most transparent to
-you.  The dates module provides several converter functions date2num
-and num2date
-
+All matplotlib date plotting is done by converting date instances into days
+since 0001-01-01 00:00:00 UTC plus one day (for historical reasons).  The
```

Comment:
What historical reasons? Is there a link to the discussion anywhere?

Reply:
http://matplotlib.org/api/dates_api.html

Reply:
Awesome. can you link to that discussion in this text to provide context.